backend/person-service/src/main/kotlin/com/nosql/personservice/repository/OfficeRepository.kt

```diff
@@ -0,0 +1,10 @@
+package com.nosql.personservice.repository
+
+import com.nosql.personservice.entity.OfficeEntity
+import org.bson.types.ObjectId
+import org.springframework.data.mongodb.repository.ReactiveMongoRepository
+import org.springframework.stereotype.Repository
+
+@Repository
+interface OfficeRepository : ReactiveMongoRepository<OfficeEntity, ObjectId> {
```

Comment:
delete empty curve brackets 

and check other repositories

Reply:
fixed